src/App.js

```diff
@@ -1,25 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
```

Comment:
- [ ] it would be better if the `App.js` should class based not a function so please make sure to do that in order to complete the requirements

Reply:
Hey @youmari Thanks for your feedback. I have gotten rid of the boilerplate. When it comes to components, the project requirements stated that it should be function based. Using class based brings me linters errors plus looking at my friends repos they all used function based components. I don't konw if that will be an issue.

Reply:
![CalculatorJs](https://user-images.githubusercontent.com/83401478/218847747-5cde28e4-01a7-46fe-8a8e-be5cd720dd8b.PNG)
-I have tried using class functions but linters won't allow me unless I disable linters